lib/sisimai/bite/email/opensmtpd.rb

```diff
@@ -0,0 +1,192 @@
+module Sisimai::Bite::Email
+  module OpenSMTPD
+    # Sisimai::Bite::Email::OpenSMTPD parses a bounce email which created by
+    # OpenSMTPD. Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/OpenSMTPD.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
```

Comment:
__[rubocop]__
* Style/MutableConstant: Freeze mutable objects assigned to constants.

<!-- Comment by [SideCI](https://sideci.com) -->